custom_components/tuya_local/devices/tuya_000004rime_weather_station.yaml

```diff
@@ -0,0 +1,586 @@
+        persist: true
+        mapping:
+          - mask: 00000000FF000000000000
+          - dps_val: 1
+            value: On
+            mask: 00000000FF000000000000
+          - dps_val: 0
+            value: Off
+            mask: 00000000FF000000000000
+      - id: 108
+        optional: true
+        type: base64
+        name: Local Temperature&Humidity
+        persist: true
+        mapping:
+          - mask: 0000000000FF0000000000
+          - dps_val: 1
+            value: On
+            mask: 0000000000FF0000000000
+          - dps_val: 0
+            value: Off
+            mask: 0000000000FF0000000000
+      - id: 108
+        optional: true
+        type: base64
+        name: Date
+        persist: true
+        mapping:
+          - mask: 000000000000FF00000000
+          - dps_val: 1
+            value: On
+            mask: 000000000000FF00000000
+          - dps_val: 0
+            value: Off
+            mask: 000000000000FF00000000
+      - id: 108
+        optional: true
+        type: base64
+        name: Week
+        persist: true
+        mapping:
+          - mask: 00000000000000FF000000
+          - dps_val: 1
+            value: On
+            mask: 00000000000000FF000000
+          - dps_val: 0
+            value: Off
+            mask: 00000000000000FF000000
+      - id: 108
+        optional: true
+        type: base64
+        name: Hourly Forecast Weather
+        persist: true
+        mapping:
+          - mask: 0000000000000000FF0000
+          - dps_val: 1
+            value: On
+            mask: 0000000000000000FF0000
+          - dps_val: 0
+            value: Off
+            mask: 0000000000000000FF0000
+      - id: 108
+        optional: true
+        type: base64
+        name: Sensor 1 Temperature&Humidity
+        persist: true
+        mapping:
+          - mask: 000000000000000000FF00
+          - dps_val: 1
+            value: On
+            mask: 000000000000000000FF00
+          - dps_val: 0
+            value: Off
+            mask: 000000000000000000FF00
+      - id: 108
+        optional: true
+        type: base64
+        name: Sensor 2 Temperature&Humidity
+        persist: true
+        mapping:
+          - mask: 00000000000000000000FF
+          - dps_val: 1
+            value: On
+            mask: 00000000000000000000FF
+          - dps_val: 0
+            value: Off
+            mask: 00000000000000000000FF
+  - entity: sensor
+    category: diagnostic
+    name: Alarm Settings
+    dps:
+      - id: 109
+        optional: true
+        type: boolean
+        name: sensor
+      - id: 109
+        optional: true
+        type: base64
+        name: Payload
+  - entity: sensor
+    category: diagnostic
+    name: Errors
+    dps:
+      - id: 113
+        optional: true
+        type: bitfield
+        name: sensor
+  - entity: sensor
+    category: diagnostic
+    name: Notifications
+    dps:
+      - id: 117
+        optional: true
+        type: bitfield
```

Comment:
Probably this is a configuration dp? if the fields are not known it should just be an extra attribute on another entity.